i18n/de/docusaurus-plugin-content-docs/current/toolbox/accessibility_indicators/closest_average.md

```diff
@@ -32,32 +36,27 @@ Described succinctly, accessibility heatmaps are a visualization representing *a
   
 :::info 
 
-Heatmaps are available in certain regions. Upon selecting a `Routing type`, a **geofence** will be displayed on the map to highlight supported regions.
+Heatmaps sind in bestimmten Regionen verfügbar. Beim Auswählen eines `Routing-Typs` wird auf der Karte ein **Geofence** angezeigt, um die unterstützten Regionen hervorzuheben.
 
 <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
   <img src={require('/img/toolbox/accessibility_indicators/heatmaps/closest_average_based/geofence.png').default} alt="Geofence for Closest-average-based Heatmaps in GOAT" style={{ maxHeight: "400px", maxWidth: "400px", alignItems:'center'}}/>
 </div> 
 
 
-If you would like to perform analyses beyond this geofence, feel free to [contact us](https://plan4better.de/en/contact/ "Contact us"). We would be happy to discuss further options.
+Wenn Sie Analysen außerhalb dieses Geofence durchführen möchten, kontaktieren Sie uns bitte [hier](https://plan4better.de/en/contact/ "Kontaktieren Sie uns"). Wir würden uns freuen, weitere Optionen mit Ihnen zu besprechen.
 
 :::
 
-## 2. Example use cases
-
- - Do residents in certain areas have longer average travel times to amenities than others?
-
- - Is there a significant difference in average travel times to amenities across different neighborhoods?
-
- - How does the average travel time to amenities vary across different modes of transport?
+## 2. Beispielanwendungsfälle
 
- - How does the average travel time vary across different types of amenities?
- 
- - Are there areas with high average travel times to amenities that could benefit from improved transport infrastructure?
+- Haben Bewohner in bestimmten Gebieten längere durchschnittliche Reisezeiten zu Annehmlichkeiten als andere?
+- Gibt es einen signifikanten Unterschied in den durchschnittlichen Reisezeiten zu Annehmlichkeiten zwischen verschiedenen Vierteln?
+- Wie variieren die durchschnittlichen Reisezeiten zu Annehmlichkeiten je nach Verkehrsmittel?
+- Wie variieren die durchschnittlichen Reisezeiten zu verschiedenen Arten von Annehmlichkeiten?
+- Gibt es Gebiete mit hohen durchschnittlichen Reisezeiten zu Annehmlichkeiten, die von einer verbesserten Verkehrsinfrastruktur profitieren könnten?
+- Wenn Standards verlangen, dass eine Mindestanzahl von Annehmlichkeiten innerhalb einer bestimmten Reisezeit zugänglich sein muss, welche Gebiete erfüllen diese Standards?
 
- - If standards require that a minimum number of amenities be accessible within a certain travel time, which areas meet these standards?
-
-## 3. How to use the indicator?
+## 3. Wie verwendet man den Indikator?
 
 <div class="step">
   <div class="step-number">1</div>
```

Comment:
@LucasLomnicki Wherever you see `<div class="content">` you can translate all the English text after this as well. Just leave the syntax and special keywords as is.

Like this:
<img width="560" alt="Screenshot 2024-05-24 at 9 06 35 AM" src="https://github.com/goat-community/goat-docs/assets/24190384/81866514-fb46-419e-81da-75df2f2dacd0">